Makefile

```diff
@@ -196,7 +195,15 @@ help:
 	@echo " - lint                             lint everything"
```

Comment:
I think a `lint-fix` would be nice as well that lints everything and fixes what it can…

Reply:
Sure, adding it.

Reply:
Added. It's only running the three linters that have a `--fix` option.